app/controllers/api/emails_controller.rb

```diff
@@ -20,7 +20,11 @@ def create
   end
 
   def draft
-    email = Email.new draft_params
+    email = Email.new draft_params.merge(
+      from: "\"#{current_user.full_name}\"<#{current_user.email}>",
+      sender: current_user.email,
+    )
```

Comment:
Instead of this, what we should actually do is set up the `Email` model to have a `User` tied to it, since we'll probably use it later.
So, after `email = Email.new draft_params`, in the next line do `email.user = current_user`.
Once that's setup, we can totally abstract the `from` and `sender` generation into inside the `Email` model.